.slugignore

```diff
@@ -1,3 +1,5 @@
 /test
 /.storybook
-/stories
\ No newline at end of file
+/stories
+./node_modules/3box
```

Comment:
I dont think we can do this, we need 3box to be available

Reply:
this was by mistake and was reverted